astroquery/mast/cutouts.py

```diff
@@ -70,21 +71,37 @@ def _parse_cutout_size(size):
                       InputWarning)
 
     # Getting x and y out of the size
+
     if np.isscalar(size[0]):
         x = size[1]
         y = size[0]
         units = "px"
+        limit_reached = True if ((x >= 30) or (y >= 30)) else False
```

Comment:
Does this 30 have to be hard-wired? (is this a server side limit?)

Reply:
Would you mind adding this description to the narrative docs, as a `Note`, too?

Reply:
Will do